src/stan_math_backend/Transform_Mir.ml

```diff
@@ -6,53 +6,42 @@ let use_opencl = ref false
 let opencl_triggers =
   String.Map.of_alist_exn
     [ ( "normal_id_glm_lpdf"
-      , ( [0; 1]
-        , [ (* Array of conditions under which to move to OpenCL *)
-            ([1], (* Argument 1 is data *)
-                  [(1, UnsizedType.UMatrix)])
-          (* Argument 1 is a matrix *)
-           ] ) )
-    ; ( "bernoulli_logit_glm_lpmf"
-      , ([0; 1], [([1], [(1, UnsizedType.UMatrix)])]) )
-    ; ( "categorical_logit_glm_lpmf"
-      , ([0; 1], [([1], [(1, UnsizedType.UMatrix)])]) )
-    ; ( "neg_binomial_2_log_glm_lpmf"
-      , ([0; 1], [([1], [(1, UnsizedType.UMatrix)])]) )
-    ; ( "ordered_logistic_glm_lpmf"
-      , ([0; 1], [([1], [(1, UnsizedType.UMatrix)])]) )
-    ; ("poisson_log_glm_lpmf", ([0; 1], [([1], [(1, UnsizedType.UMatrix)])]))
-    ]
+      , [ (* Array of conditions under which to move to OpenCL *)
+          [(1, UnsizedType.UMatrix)]
+        (* Argument 1 is a matrix *)
+         ] )
+    ; ("bernoulli_logit_glm_lpmf", [[(1, UnsizedType.UMatrix)]])
+    ; ("categorical_logit_glm_lpmf", [[(1, UnsizedType.UMatrix)]])
+    ; ("neg_binomial_2_log_glm_lpmf", [[(1, UnsizedType.UMatrix)]])
+    ; ("ordered_logistic_glm_lpmf", [[(1, UnsizedType.UMatrix)]])
+    ; ("poisson_log_glm_lpmf", [[(1, UnsizedType.UMatrix)]]) ]
```

Comment:
In the old format, we also listed which arguments were matrix_cl and which had to be data. Those will no longer be requirements. matrix_cl overloads will all accept matrix_cl, var_value<matrix_cl> or scalar (where applicable). And no data requirements are required anymore.

For some functions, we only move to GPU for a subset of the signatures (others are slower).

Reply:
Sweet. 

> For some functions, we only move to GPU for a subset of the signatures (others are slower).

What's an example of this?